README.md

```diff
@@ -47,4 +47,4 @@ Open up your browser to http://localhost:1313 to view the site.
 
 ## Thank you!
 
-Kubernetes thrives on community participation, and we really appreciate your contributions to our site and our documentation!
+Kubernetes thrives on community participation, and we really appreciate your contributions to our sites and our documentations!
```

Comment:
These should both stay singular, because we only have one site and one set of documentation. Please revert these.